tasks/pruner-present.yml

```diff
@@ -28,6 +28,7 @@
         - builds
         - deployments
         - images
+      when: item not in appuio_pruner_disabled_jobs
```

Comment:
It may make sense to have an extra step before this one which verifies that all entries in `appuio_pruner_disabled_jobs` are actually jobs that we know of, to avoid unwelcome surprises stemming from typos or similar.

Reply:
There is  now a step to validate whether jobs in `appuio_pruner_disabled_jobs` are valid.